modern/src/common/attributes/useCommonServerAttributes.js

```diff
@@ -0,0 +1,8 @@
+import { useMemo } from 'react';
+
+export default (t) => useMemo(() => ({
```

Comment:
This file should be called `useServerAttributes` because it doesn't have any common attributes.

Reply:
Agreed, it probably wont be that common. Changed it in [60995ed](https://github.com/traccar/traccar-web/pull/970/commits/60995ed858af7da0493c4fd84f38f3c7709e2474)